packages/plugin-core/src/components/views/TutorialPreviewLinkHandler.ts

```diff
@@ -0,0 +1,39 @@
+import { TutorialEvents } from "@dendronhq/common-all";
+import { AnalyticsUtils } from "../../utils/analytics";
+import { LinkType, PreviewLinkHandler } from "./PreviewLinkHandler";
+
+export class TutorialPreviewLinkHandler extends PreviewLinkHandler {
+  /**
+   *  set of tutorial note ids that we will allow tracking of link clicked events.
+   */
+  private _allowedIds = new Set<string>(["c1bs7wsjfbhb0zipaywqfbg"]);
```

Comment:
While I'm still relying on the hacky impl for workspace note telemetry, can we add this ID to `_extension.ts`? https://github.com/dendronhq/dendron/blob/55630a47455e960b01a7499de108d16a7ef832f1/packages/plugin-core/src/_extension.ts#L269